pkg/spec/cluster.go

```diff
@@ -193,6 +193,8 @@ type ClusterStatus struct {
 
 	// Size is the current size of the cluster
 	Size int `json:"size"`
+	// HealthyEndpoints are the 'PodIP:Port' endpoints passing the HTTP GET liveliness probe
+	HealthyEndpoints []string `json:"healthyEndpoints"`
```

Comment:
I think our final conclusion is to use `ReadyPods []string`, probably also provide info for `UnreadyPods []string` as well. See https://github.com/coreos/etcd-operator/pull/865#discussion_r104585695